intellij-plugin-structure/structure-intellij/src/main/java/com/jetbrains/plugin/structure/intellij/problems/LevelRemappingPluginCreationResultResolver.kt

```diff
@@ -58,6 +58,10 @@ class LevelRemappingPluginCreationResultResolver(private val delegatedResolver:
   }
 
   private fun remapPluginProblemLevel(pluginProblem: PluginProblem): PluginProblem? {
+    return remapPluginProblemLevel(pluginProblem, unwrapRemappedProblems)
+  }
+
+  private fun remapPluginProblemLevel(pluginProblem: PluginProblem, unwrapRemappedProblems: Boolean): PluginProblem? {
```

Comment:
can't understand why you need this

Reply:
There might be a chain of multiple level remapping resolvers. The majority of instances do not need to remap an already remapped problem, but there are cases when this is necessary.